### ISSUE TITLE
Gem::LoadError sqlite3

### ISSUE THREAD

**Issue by amrapalijz** [slrbl/human-in-the-loop-machine-learning-tool-tornado#1]
When I execute the `rake db:migrate` command I get:

`Amrapalis-MBP:tornado-active-learning amrapaliz$ rake db:migrate
rake aborted!
Gem::LoadError: Specified 'sqlite3' for database adapter, but the gem is not loaded. Add `gem 'sqlite3'` to your Gemfile (and ensure its version is at the minimum required by ActiveRecord).`

The Gemfile already has sqlite3. Which version should we add?

**Comment by slrbl** [comment]
Try the following:
`rm Gemfile.lock`
add gem 'sqlite3' to Gemfile
`bundle install`

**Comment by amrapalijz** [comment]
That didn't work. 
This is the Gemfile I should edit, right: https://github.com/slrbl/tornado-active-learning/blob/master/Gemfile ? It already contains gem 'sqlite3'. Does it need a version?

**Comment by slrbl** [comment]
Yes, I think that you need to verify your active record version and find the suitable version of sqlite3.

**Comment by amrapalijz** [comment]
Ah ok I updated my Gemfile to
`gem 'sqlite3', '~> 1.3.6'`
and now it works.
